src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import MuiDialog, { DialogProps as MuiDialogProps } from '@mui/material/Dialog';
+
+export interface DialogProps extends MuiDialogProps { }
+
+const Dialog = ({
+  children,
+  sx,
+  ...props
+}: DialogProps): JSX.Element => (
+  <MuiDialog
+    sx={{
+      border: 'inherit',
+      borderRadius: '16px',
+      ...sx,
+    }}
+    {...props}
```

Comment:
rather than splitting out `sx` like this, we should just make use of the `styled` API for the base styles, so that we can just pass props directly. (see `Button`, for example)